Add code block markdown to README.md

pdelboca opened the pull request:
Just add some code blocks to the documentation.

shevron commented:
Markdown allows code blocks by either 4 space indentiation (as I used) or by ````` markers. I'm OK with switching to ```` markers (maybe they don't work in all situations?) but then we should deindent the blocks.

pdelboca commented:
Ok! For some reason the 4 space indentation doesn't like bullet points.

If I do:
```
* Bullet point
    Four space indentation
```
It doesn't work, but if I do:
```
No Bullet point
    Four space indentation
```
It works. I added the ` ``` ` markups and removed the indentation.
